test/parse.js

```diff
@@ -13,6 +13,10 @@ test('query strings starting with a `&`', t => {
 	t.deepEqual(queryString.parse('&foo=bar&foo=baz'), {foo: ['bar', 'baz']});
 });
 
+test('query strings ending with a `&`', t => {
+	t.deepEqual(queryString.parse('foo=bar&'), {foo: 'bar'});
```

Comment:
Can you also do a test for `foo=bar&&&`?

Reply:
Sure, good idea. It's added